README.md

```diff
@@ -573,3 +573,48 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+
+### renderXXX
+
+For better navigation, use `render` prefix for any method who returns JSX or component.  
+Of course it is possible to use another prefixes like `get` or even `display`, but in terms of React we recommend to use `renderXXX`.
```

Comment:
I suggest to make using `render` strict without alternative

Reply:
I too. Do you suggest to remove this phrase?

Reply:
done